partner_communication_switzerland/models/staff_notifications_settings.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Samuel Fringeli <samuel.fringeli@me.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import api, models, fields
+
+
+class StaffNotificationSettings(models.TransientModel):
+    """ Settings configuration for any Notifications."""
+    _inherit = 'staff.notification.settings'
+
+    # Users to notify after Disaster Alert
+    invalid_mail_notify_ids = fields.Many2many(
+        'res.partner', string='Invalid mail ids',
```

Comment:
You should add parameters relation, column1 and column2 to avoid conflicts with other many2many relations. See https://www.odoo.com/documentation/10.0/reference/orm.html#relational-fields for field reference.
Change string to "Invalid mail" only.